docs/TestCoverage.md

```diff
@@ -3895,25 +3895,26 @@ expect(node, inputs=[data_0, data_1], outputs=[result],
 ### Mod
 There are 4 test cases, listed as following:
```

Comment:
Looks like TestCoverage.md was not updated bc the changes you made in python script are not refected.